src/components/sidebar/style.module.css

```diff
@@ -21,7 +21,7 @@
 	color: #fff;
 	fill: rgba(255, 255, 255, 0.6);
 	cursor: pointer;
-	z-index: 560;
+	z-index: 460;
```

Comment:
I think this was a typo.

The slide-out sidebar has a `z-index` of `450`, and the slide-down header has a `z-index` of `500`. As such, when the header is expanded, there's really no reason for the sidebar toggle button to be displayed -- you're toggling something that can't be seen.

`z-index` of `460` allows it to still sit on top of the sidebar when expanded but hidden behind the header if it is expanded.